astro/src/content/docs/get-started/use-cases/app-suite.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: App Suite
+description: Let your users log in once and then transparently be logged into others.
+section: get started
+subcategory: use cases
+navOrder: 20
+---
+import ApplicationSwitchDiagram from 'src/diagrams/docs/get-started/use-cases/app-suite/application-switch.astro';
+import AppSuiteDescription from 'src/content/docs/get-started/use-cases/_app-suite-description.mdx';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import InlineField from 'src/components/InlineField.astro';
+
+## Overview
+
+<AppSuiteDescription />
+
+## Problem 
+
+You have multiple applications that you want to share across a common user base.
+
+These applications may:
+
+* be hosted at different domains or under different paths
+* use SAML or OIDC for authentication
+* be custom, commercial, or open source
+* be written using different technology stacks
+
+You want to let your users log in once, and switch between applications without having to log in again. Users have different roles for each application.
+
+## Solution
+
+With the app suite use case, after the user logs in to one application, they are logged in to all of them.
+
+You'll also have one customer profile and can view their login activity across the applications.
+
+## Prerequisites
+
+You have configured your application or applications to [delegate authentication to FusionAuth](/docs/get-started/use-cases/auth-service) via the Authorization Code grant and the hosted login pages.
```

Comment:
```suggestion
You have configured your application or applications to [delegate authentication to FusionAuth](/docs/get-started/use-cases/auth-service) via the Authorization Code grant and the hosted login pages.  In other words you are using OAuth2, our built in workflows, and JWT's to authenticate a user.
```

Reply:
I feel like this is maybe more detail that is needed, given the link.